packages/openactive-integration-tests/test-data-generator/test-data-generator.js

```diff
@@ -98,14 +102,24 @@ const categoryOrFeature = /** @type {string} */(categoryOrFeatureUntyped); // ya
     // Broker is not running, so we override it with seller config directly from the config file.
     const seller = getSellerConfigFromSellerCriteria(sellerCriteria, SELLER_CONFIG);
     for (const [opportunityCriteria, numOpportunitiesRequired] of opportunityCriteriaRequirements) {
-      for (const opportunityType of IMPLEMENTED_OPPORTUNITY_TYPES) {
-        numberOfItems += numOpportunitiesRequired;
-        const testInterfaceOpportunity = createTestInterfaceOpportunity(opportunityType, opportunityCriteria, seller['@id'], seller['@type']);
-        itemListElement.push({
-          '@type': 'ListItem',
-          'test:numberOfInstancesInDistribution': numOpportunitiesRequired,
-          item: dissoc('@context', testInterfaceOpportunity),
-        });
+      for (const bookingFlow of IMPLEMENTED_BOOKING_FLOWS) {
+        for (const opportunityType of IMPLEMENTED_OPPORTUNITY_TYPES) {
+          // TODO this needs to take into account FeatureHelper.skipOpportunityTypes/skipBookingFlows
```

Comment:
it really does. Not urgently problematic though as this only means it will ask for too much test data rather than too little